zebra/zebra_rib.c

```diff
@@ -4580,18 +4580,34 @@ int rib_add_multipath(afi_t afi, safi_t safi, struct prefix *p,
 
 		ifp = if_lookup_prefix(p, re->vrf_id);
 		if (ifp) {
+			struct zebra_vrf *zvrf = zebra_vrf_lookup_by_id(re->vrf_id);
```

Comment:
maybe a nit: since we're now more sensitive to variable shadowing, it might be safer to put declarations at the top of the function, and try to avoid "inner" declarations. I know we've got a lot of that though...

Reply:
ha!  we've been asking for years that people allocate variables as close as possible to limit scope.  If there is some sort of shadow warning I would prefer that the compiler bitch and moan to us as a way to handle this :)